data/volunteerStorage.json

```diff
@@ -0,0 +1,57 @@
+{
+  "volunteers" : [ {
+    "name" : "Alexis Yeoh",
+    "phone" : "87438807",
+    "email" : "alexyeoh@example.com",
+    "skills" : [ "packer", "teacher" ],
+    "assignedEvents" : [ ]
+  }, {
+    "name" : "Bernice Yu",
+    "phone" : "99272758",
+    "email" : "berniceyu@example.com",
+    "skills" : [ "smart", "intelligent" ],
+    "assignedEvents" : [ ]
+  }, {
+    "name" : "Charlotte Oliveiro",
+    "phone" : "93210283",
+    "email" : "charlotte@example.com",
+    "skills" : [ "mechanic" ],
+    "assignedEvents" : [ ]
+  }, {
+    "name" : "David Li",
+    "phone" : "91031282",
+    "email" : "lidavid@example.com",
+    "skills" : [ "cook" ],
+    "assignedEvents" : [ ]
+  }, {
+    "name" : "Irfan Ibrahim",
+    "phone" : "92492021",
+    "email" : "irfan@example.com",
+    "skills" : [ "muscular", "strong" ],
```

Comment:
Is muscular and strong a skill hahahahha

Reply:
my skill 👍